index.js

```diff
@@ -105,7 +174,21 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies,genre) {
+  if(movies.length === 0){
+    return []
+  }
+
+  let arr = []
+
+  for(movie of movies){
+    let currGenres = movie.genre.toLowerCase()
+    if (currGenres.includes(genre.toLowerCase())){
+      arr.push(movie)
+    }
+  } 
+  return arr
+}
```

Comment:
great work with the .`toLowerCase()` - would this work if we used `toUpperCase()` instead?